bin/lib/absorb.js

```diff
@@ -323,6 +326,20 @@ function getDataFromURL(feedInfo){
 		})
 	;
 
+	const currentTime = new Date();
+	
+	// If it's between 13:00 and 13:10, and we have not sent a problem report
```

Comment:
let's redo it so we record the ids (or lack thereof) of items which fail to parse: if its a new id, email immediately. This will trigger once per day (per item) after that if not fixed when the heroku instance restarts.